README.md

```diff
@@ -282,8 +282,9 @@ All externally visible exceptions are in the `\MaxMind\Exception` namespace.
 The possible exceptions are:
 
 * `InvalidInputException` - This will be thrown when the `->report()` method is
-  called with invalid input data or when the required `ip_address` or `tag`
-  fields are missing.
+  called with invalid input data or when the required fields are missing. The
+  required fields are `tag` and one or more of the following: `ip_address`,
+  `maxmind_id`, `minfraud_id`, or `transaction_id`.
```

Comment:
```suggestion
  called with invalid input data or when the required arguments are null. The
  required arguments are `tag` and one or more of the following: `ipAddress`,
  `maxmindId`, `minfraudId`, or `transactionId`.
```

This one is my fault. I should have updated this when I switched to named arguments as the preferred method.